src/EnergyPlus/WindowManager.cc

```diff
@@ -6070,7 +6070,7 @@ namespace WindowManager {
 			TInRadFr = TInRad * root_4( ( 1.0 + 0.5 * ProjCorrFrIn ) / ( 1.0 + ProjCorrFrIn ) );
 			FrameCon = SurfaceWindow( SurfNum ).FrameConductance;
 			HInRad = 0.5 * SurfaceWindow( SurfNum ).FrameEmis * sigma * pow_3( TInRadFr + SurfaceWindow( SurfNum ).FrameTempSurfIn + TKelvin );
-			HInConvFr = 0.0;
+			HInConvFr = HInConv;
```

Comment:
I have always thought that the original design intent in EnergyPlus for window frames was never consistent with the usual approach taken when determining assembly U-factors.  (A complete rework has long been needed in this area to properly handle frames, edge-of-glass, dividers and assembly U-factor had been on the enhancment list for a long long time.)   

I believe the idea that is showing up here was that surrounding base surface would the part of the "frame" and what is calculated in E+ as "frame" is just the projecting perpendicular surface.  So if no projection, no frame and no extra convection.